.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,23 @@
+## Description
+
+Please, describe the feature this PR is introducing.
```

Comment:
This can also point to an issue describing the feature. No need to duplicate description, provided the issue describes the feature well enough.

Reply:
I have not assumed that all PR's are open by team members, and as such I would not expect someone that open a feature PR to necessarily have also opened an issue.

Reply:
fair enough.